lib/test.js

```diff
@@ -80,7 +80,11 @@ function wrapAssertFn(assertFn) {
     var err = assertFn(res);
     if (err && err.stack) {
       badStack = err.stack.replace(err.message, '').split('\n').slice(1);
-      err.stack = [err.toString(), savedStack, '----', badStack].flat().join('\n');
+      err.stack = [err.toString()]
+        .concat(savedStack)
+        .concat('----')
+        .concat(badStack)
+        .join('\n');
```

Comment:
Can you use `push` instead?  I believe it is much faster than `concat`.

Reply:
The idea of using `concat` is because the `savedStack` and `badStack` is an array.
push will require hack using `array.push.apply(array, savedStack)`. Ok with that?